http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppUsageComponent.scala

```diff
@@ -14,6 +14,7 @@ import org.typelevel.log4cats.Logger
 import java.sql.SQLDataException
 import java.time.Instant
 import scala.concurrent.ExecutionContext
+import scala.util.control.NoStackTrace
 
 class AppUasageTable(tag: Tag) extends Table[AppUsageRecord](tag, "APP_USAGE") {
```

Comment:
@Qi77Qi while you're in here this typo (AppUasageTable) might be a nice fix too

Reply:
@mlilynolting  thanks for checking my spelling :) @kyuksel used to be my "type checker" lol

Reply:
😊 Thanks for taking over the torch @mlilynolting 🔥